src/components/ResultCard/ResultCard.tsx

```diff
@@ -1,31 +1,71 @@
-import React, { KeyboardEvent } from 'react';
+import React, { KeyboardEvent, useContext } from 'react';
 import { BrowseResultData } from '../../types';
 import ResultCtaButton from '../ResultCtaButton/ResultCtaButton';
+import QuizContext from '../CioQuiz/context';
 
 interface ResultCardProps {
   result: Partial<BrowseResultData>;
   addToCartCallback: (clickedResult: Partial<BrowseResultData>) => any;
   clickItemCallback?: (clickedResult: Partial<BrowseResultData>) => any;
   salePriceKey?: string;
   regularPriceKey?: string;
+  resultPosition: number;
 }
 
 export default function ResultCard(props: ResultCardProps) {
-  const { result, addToCartCallback, clickItemCallback, salePriceKey, regularPriceKey } = props;
+  const {
+    result,
+    addToCartCallback,
+    clickItemCallback: customClickItemCallback,
+    salePriceKey,
+    regularPriceKey,
+    resultPosition,
+  } = props;
+  const { resultsResponse } = useContext(QuizContext);
+  const { cioClient } = useContext(QuizContext);
```

Comment:
🤔 Is there any reason why we have these on separate lines?

Reply:
Not really 🤔 Similar code was on separate lines on existing code so I didn't really think why and followed that. I think they can be on the same line. Will update